### PR TITLE
fixing sklearn doc and transformers

### LINKED ISSUE #568: DOC keras + sk transformer in README.md
We forgot to write about them...

### PR DESCRIPTION
**Pull request by edouardoyallon**
Closes #568

### CODE REVIEW

**Comment by edouardoyallon** [comment]
```
/home/circleci/repo/doc/source/gallery_1d/plot_real_signal.rst:17: WARNING: Unexpected section title or transition.

-------------
/home/circleci/repo/doc/source/gallery_1d/plot_real_signal.rst:19: WARNING: Missing matching underline for section title overline.
```
it could be fixed too

File: tests/scattering2d/test_keras_scattering2d.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import tensorflow as tf
```
Comment (janden) [inline tests/scattering2d/test_keras_scattering2d.py]: Move to separate commit.

File: README.md [hunk 2]
```diff
@@ -102,6 +102,25 @@ from kymatio.tensorflow import Scattering2D
 scattering = Scattering2D(J=2, shape=(32, 32))
 ```
 
+## Keras
+
+After installing the latest version of `keras`, you can call `Scattering2D` as a `tf.Module` via for instance:
```
Comment (janden) [inline README.md]: It's just TF. You don't install Keras separately anymore. Also `tf.Module` should be just “Keras `Layer`”.

File: README.md [hunk 3]
```diff
@@ -102,6 +102,25 @@ from kymatio.tensorflow import Scattering2D
...
+scat = Scattering2D(J=2)(inputs)
+```
+
+## Scikit-learn
+
+After installing the latest version of `sklearn`, you can call `Scattering2D` as a `Transformer` via for instance:
+
+```
+from kymatio.sklearn import Scattering2D
+scat = ScatteringTransformer2D(2, (32, 32))
```
Comment (janden) [inline README.md]: Just `Scattering2D`.
